README.md

```diff
@@ -82,6 +82,7 @@ See [all supported options in the documentation at  `docs/options.md`](docs/opti
 | gherkin | `Gherkin` |`.feature` | [`Gherkin formatter`](https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/gherkin.coffee) (Default) |
 | GLSL | `C`, `opencl`, `GLSL` |`.vert`, `.frag` | [`clang-format`](https://clang.llvm.org/docs/ClangFormat.html) (Default) |
 | Go | `Go` |`.go` | [`gofmt`](https://golang.org/cmd/gofmt/) (Default) |
+| Golang Template | `HTML (Go)`, `Go Template` |`.gohtml` | [`Pretty Diff`](https://github.com/prettydiff/prettydiff) (Default) |
```

Comment:
There should be more documentation updates than just this.

Please run `npm run docs` to generate the updated documentation. You should see changes in `docs/` directory as well as in the `README.md`.